.travis.yml

```diff
@@ -33,6 +33,8 @@ script:
   - ./test.sh
   # Run Travis-only tests
   - ./travis_only_test.sh
+  # Run gui unit tests
```

Comment:
Shouldn't these things be in the opposite order - first build, then test?

Reply:
These are unit tests and don't require a build to run. Thus, test first, then build.

Reply:
In yesterday's software meeting we discussed moving everything to CircleCI. My intention is to remove the Travis config after confirming we have feature parity. Does this sound okay to you @hhff ?

Reply:
Yup! I think the main blocker is #182